pyfoxsi/src/pyfoxsi/response/response.py

```diff
@@ -34,24 +36,33 @@ class Response(object):
     >>> resp = Response()
     >>> resp1 = Response(shutter_state=1)
     """
-    def __init__(self, shutter_state=0):
+    def __init__(self, shutter_state=0, configuration=1):
         path = os.path.dirname(pyfoxsi.__file__)
         for i in np.arange(3):
             path = os.path.dirname(path)
         path = os.path.join(path, 'data/')
-        filename = 'effective_area_per_shell.csv'
+        filename = 'effective_area_per_module.csv'
         effarea_file = os.path.join(path, filename)
-        self._eff_area_per_shell = pd.read_csv(effarea_file, index_col=0)
+        optics_effective_area = pd.read_csv(effarea_file, index_col=0, skiprows=4)
+        optics_effective_area = optics_effective_area[optics_effective_area.columns[configuration-1]]
+        pyfoxsi.focal_length = 15 * u.m
+        pyfoxsi.number_of_telescopes = 3
+        if configuration > 1:
+            pyfoxsi.focal_length = 10 * u.m
+        if configuration > 2:
+            pyfoxsi.number_of_telescopes = 2
```

Comment:
The style of these `if` statements is prone to confusion without any clarifying comments, and are likely to break when someone else touches them.  Also, I'd recommend using `>=` rather than '>` comparisons.